workflow/Snakefile

```diff
@@ -74,6 +84,57 @@ rule create_results_directory:
         touch {output.TEMP_}
         """
 
+rule plot_sequence_logos:
+    """
+    Create directories for the results
+    """
+    input:
```

Comment:
Could you also link to the rule `create_output_directories` here? we need a dir for results before each of the other rules. We need a place to write to.

Reply:
Yes, you are right, linked!